README.md

```diff
@@ -2,10 +2,73 @@
 
```

Comment:
Generally to whole project, I still don't see any meaningfull folder/file structure, just default from react